.circleci/config.yml

```diff
@@ -1,10 +1,8 @@
 version: 2.1
 
-orbs:
```

Comment:
not currently running cypress tests on the repo. the old version of the site had ineffective tests and now there isn't anything custom here so we don't need to test docusaurus in this case.